desktop-app/app/menu.js

```diff
@@ -68,6 +69,7 @@ export default class MenuBuilder {
             webPreferences: {
               devTools: false,
               nodeIntegration: true,
+              additionalArguments: [JSON.stringify(getAllShortcuts())]
             },
```

Comment:
@jjavierdguezas Should we also look at registering the `accelerators` defined below in this file via the shortcuts-manager module?

Reply:
maybe.. if you look into the hotkeys dialog HTML file I handle those "static" shortcuts separately from the "dynamic" ones. Do you want to store them all in the manager?